src/terra-dev-site/doc/application/reference.1/components/DisclosureManager.doc.mdx

```diff
@@ -0,0 +1,12 @@
+# DisclosureManager
+
+Manages application disclosures. See terra-disclosure-manager for general usage details.
+
+## Dismiss Callback Stack
+Content disclosed with terra-application's DisclosureManager can be dismissed from outside of a
```

Comment:
Let me know if there's a better place to put this information or if you want it documented at all.

Reply:
I'd vote no docs at all. I don't want anyone consuming this outside of our internal usage.

Reply:
Removed https://github.com/cerner/terra-application/pull/29/commits/3071d812782e671283d07dabc9e7cbeefb5a4b6b